internal/policygen/policygen.go

```diff
@@ -38,7 +40,7 @@ const (
 // RunArgs is the struct representing the arguments passed to Run().
 type RunArgs struct {
 	ConfigPath string
-	StatePath  string
+	StatePaths string
```

Comment:
why not make this a list of string and do the split in the caller?

Reply:
I'm ok either way. I feel using a comma separated list makes the command line a bit more compact and is what `--templates` is doing tin the tfengine command line.

Reply:
tfengine seems to do it in the main as well: https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/cmd/tfengine/main.go#L67

I would vote for doing this in main. Seems odd to have a  StatePaths (plural) that's a string (singular). list of strings is more explicit.

Reply:
Done. Thought you were referring to custom flag. Moved to main.go